config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w(
+  .ruby-version
+  .rbenv-vars
+  tmp/restart.txt
+  tmp/caching-dev.txt
+).each { |path| Spring.watch(path) }
```

Comment:
do we really need spring?

Reply:
I really enjoy use it in dev mode. quite fast/stable. why not? :)

Reply:
ah ok